python-minimal-streamlit/.renku/renku.ini

```diff
@@ -0,0 +1,5 @@
+[renku "interactive"]
+default_url = /streamlit
+lfs_auto_fetch = true
+cpu_request = 2
+mem_request = 8G
```

Comment:
Maybe it's better to just leave this as default values?

Reply:
I'd prefer to keep these values as I use streamlit mainly for visualizing and the default values were so far not sufficient for that purpose.

Reply:
right, but this is a template for a generic user and this might not be expected. We'll be adding a feature in the UI that will make it much easier (and quicker) to modify these settings for each project so it won't be as big of a hassle soon.